carbonmark/components/pages/Project/Purchase/PurchaseInputs.tsx

```diff
@@ -0,0 +1,186 @@
+import { Anchor } from "@klimadao/lib/components";
+import { urls } from "@klimadao/lib/constants";
+import { formatUnits } from "@klimadao/lib/utils";
+import { t, Trans } from "@lingui/macro";
+import HelpOutline from "@mui/icons-material/HelpOutline";
+import { Dropdown } from "components/Dropdown";
+import { InputField } from "components/shared/Form/InputField";
+import { Text } from "components/Text";
+import { formatToPrice } from "lib/formatNumbers";
+import { carbonmarkPaymentMethodMap } from "lib/getPaymentMethods";
+import { LO } from "lib/luckyOrange";
+import { CarbonmarkPaymentMethod, Listing } from "lib/types/carbonmark";
+import Image from "next/legacy/image";
+import { useRouter } from "next/router";
+import { FC } from "react";
+import { SubmitHandler, useFormContext } from "react-hook-form";
+import * as styles from "./styles";
+import { FormValues } from "./types";
+
+type Props = {
+  onSubmit: (values: FormValues) => void;
+  listing: Listing;
+  values: null | FormValues;
+  balance: string | null;
+};
+
+export const PurchaseInputs: FC<Props> = (props) => {
+  const { locale } = useRouter();
+
+  const { register, handleSubmit, formState, control, clearErrors } =
+    useFormContext<FormValues>();
+
+  const onSubmit: SubmitHandler<FormValues> = (values: FormValues) => {
+    LO.track("Purchase: Continue Clicked");
+    props.onSubmit(values);
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <div className={styles.inputsContainer}>
+        <InputField
+          id="listingId"
+          inputProps={{
+            type: "hidden",
+            ...register("listingId"),
+          }}
+          label={"listing ID"}
+          hideLabel
+        />
+
+        <div className={styles.amountLabel}>
+          <Text>{t`How many tonnes of carbon do you want to buy?`}</Text>
+          <Text t="body3">
+            <Trans>Available:</Trans> {formatUnits(props.listing.leftToSell)}
```

Comment:
Interpolate the value into the string because other languages might format this differently

```suggestion
<Trans>Available: {formatUnits(props.listing.leftToSell)}</Trans> 
```

Reply:
done